PresentScreenings/Views/ColorView.cs

```diff
@@ -214,15 +285,101 @@ public static void DrawOnDemandAvailabilityStatus(CGContext context, OnDemandScr
                     {
                         needlePath.AddLines(new CGPoint[]
                         {
-                    new CGPoint(i*wPeriod, y + h*1/16),
-                    new CGPoint(i*wPeriod, y + h*3/16)
+                            new CGPoint(i*wPeriod, y + h*1/16),
+                            new CGPoint(i*wPeriod, y + h*3/16)
                         });
                     }
                     context.AddPath(needlePath);
                 }
             }
             context.DrawPath(CGPathDrawingMode.FillStroke);
         }
+
+        public static void DrawWarningMiniature(CGContext context, nfloat side)
+        {
+            if (side >= _clickpadMinSideToDrawAll)
+            {
+                nfloat x = side * 1 / 16;
+                nfloat y = side * 4 / 16;
+                context.SetLineWidth(1);
+                NSColor.Black.SetStroke();
+                WarningBackgroundColor.SetFill();
+                var trianglePath = new CGPath();
+                trianglePath.AddLines(new CGPoint[]{
+                    new CGPoint(x + side*1/16, y + side*1/16),
+                    new CGPoint(x + side*4/16, y + side*7/16),
+                    new CGPoint(x + side*7/16, y + side*1/16)
+                });
+                trianglePath.CloseSubpath();
+                context.AddPath(trianglePath);
+                context.DrawPath(CGPathDrawingMode.FillStroke);
+                var barPath = new CGPath();
+                barPath.AddLines(new CGPoint[]{
+                    new CGPoint(x + side*4/16, y + side*2/16),
+                    new CGPoint(x + side*4/16, y + side*5/16)
+                });
+                barPath.CloseSubpath();
+                context.AddPath(barPath);
+                context.DrawPath(CGPathDrawingMode.Stroke);
+            }
+        }
+
+        public static void InitializeCoreText(CGContext context, bool selected, nfloat side)
+        {
+            var yShift = 3;
+            context.TranslateCTM(-1, yShift);
```

Comment:
Please let it be consequent, either magic numbers (not preferred), or constant names.
Please move constant names to region `Constants`.

Reply:
Done.